### PR TITLE
changes due to dynamic threshold function evaluation

### PR DESCRIPTION
**Pull request by tannerlex**

### CODE REVIEW

File: anomaly_utilities.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import os
 import numpy as np
 import pandas as pd
-from scipy.stats import norm
+from scipy.stats import norm, t
```
Comment (tannerlex) [inline anomaly_utilities.py:13]: remove t

File: anomaly_utilities.py [hunk 2]
```diff
@@ -228,7 +228,7 @@ def xfade(xfor, xbac):
     return x
 
 
-def set_dynamic_threshold(data, alpha, window_sz):
+def set_dynamic_threshold(residuals, alpha, window_sz):
     """Determines a threshold based on the local confidence interval, 
     considering the data looking forward and backward window_sz steps.
     data is a series like object
```
Comment (tannerlex) [inline anomaly_utilities.py:234]: update "data" to be "residuals"

**Review by tannerlex — COMMENTED** [review]
I left a couple of nit-picking comments

**Review by tannerlex — COMMENTED** [review]
also you could remove the comments inside ARIMA_detect.py in the function set_threshold where it discusses a dynamic threshold

File: anomaly_utilities.py (hunk 2 above)
Comment (AmberSJones) [reply anomaly_utilities.py:234]: updated the comments in the function documentation

**Pull request merged**